src/library/scala/collection/TraversableOnce.scala

```diff
@@ -402,8 +402,8 @@ trait TraversableOnce[+A] extends Any with GenTraversableOnce[A] {
 
 
 object TraversableOnce {
-  implicit def alternateImplicit[A](trav: TraversableOnce[A]) = new ForceImplicitAmbiguity
-  implicit def flattenTraversableOnce[A, CC[_]](travs: TraversableOnce[CC[A]])(implicit ev: CC[A] => TraversableOnce[A]) =
+  implicit def alternateImplicit[A](trav: TraversableOnce[A]): ForceImplicitAmbiguity = new ForceImplicitAmbiguity
```

Comment:
It's the return of Java boilerplate!  Are we really advocating this as best practice (explicit return type when you are literally just calling the constructor of that very type)?